src/CommunityToolkit.Maui.UnitTests/Alerts/SnackBar_Tests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using CommunityToolkit.Maui.Alerts.Snackbar;
+using Xunit;
+
+namespace CommunityToolkit.Maui.UnitTests.Alerts;
+
+public class Snackbar_Tests : BaseTest
```

Comment:
Let's make sure we cover the entire `ISnackbar` and `Snackbar` APIs.

Here's the list of APIs that we still need tests for:
- `ISnackbar.Action`
- `ISnackbar.ActionButtonText`
- `ISnackbar.Anchor`
- `ISnackbar.Duration`
- `ISnackbar.Text`
- `ISnackbar.VisualOptions`
- `Snackbar.Make`
- `DisplaySnackbar()`